diktat-gradle-plugin/pom.xml

```diff
@@ -100,7 +99,8 @@
                                 <argument>-PjunitVersion=${junit.version}</argument>
                                 <argument>-PjacocoVersion=${jacoco.version}</argument>
                                 <argument>-S</argument>
-                                <argument>${gradle.exclude.check}</argument>
+                                <!-- This argument can be set to excluded task in the specific build profile -->
+<!--                                <argument></argument>-->
```

Comment:
new version of exec-maven-plugin treats empty arguments as string with whitespace " " and uses `java.lang.ProcessBuilder` which fails with such command

Reply:
The diff is `https://github.com/mojohaus/exec-maven-plugin/compare/exec-maven-plugin-3.0.0...exec-maven-plugin-3.1.0`
Tried to reproduce, but it does work for test project: https://github.com/nulls/exec-mojo-processor-builder

Reply:
Problem is in <argument> (only this change I see in exec plugin during debugging), removed for now